src/main/java/org/fundacion/jala/converter/view/MainInterface.java

```diff
@@ -10,35 +10,59 @@
  */
```

Comment:
You should put the full name of the author

Reply:
Solved!